webpack.config.js

```diff
@@ -18,37 +18,45 @@ const { getIfUtils, removeEmpty } = require( 'webpack-config-utils' );
 
 
 module.exports = ( env ) => {
-  const { ifProd, ifNotProd, ifTest, ifNotTest } = getIfUtils( env );
+  const { ifProd, ifNotProd, ifTest, ifDev, ifSite } = getIfUtils( env, ['prod', 'test', 'dev', 'site'] );
+
+  function _packagePrefix() {
+    return './packages' + ( env.element ? `/${env.element}` : '' );
+  }
+
+  function _entryPoint() {
```

Comment:
again nit pick.

name functions by what they do.

This one returns entry point, so `function getEntryPointConfig() {` is definitely more appropriate.

Points to do:
- properly resolve paths via `resolve` instead of string concat
- you dont need to use strings for object keys

Points to consider:
- also we could make it pure and extract it from direct config

```js
function getEntryPointConfig(basePath, {isTest,isProd}={}) {
   if( isTest ){
     return {
       test: resolve(basePath, '/index.test.ts'),
       'test-helpers': './test-helpers.ts',
     };
   }
   if( isProd ){
      return /*...*/
   }
   return /*...*/

   
}
```

then:

```js
module.exports = ( env ) => {
  const { ifProd, ifNotProd, ifTest, ifDev, ifSite } = getIfUtils( env, ['prod', 'test', 'dev', 'site'] );
  const packagePath = resolve('./packages', env.element || '' );
  const entry = getEntryPointConfig(packagePath,{isTest:isTest(),isProd:isProd()});
  // ...
}
```